common/src/main/java/eu/kevin/common/helpers/WebFrameHelper.kt

```diff
@@ -0,0 +1,50 @@
+package eu.kevin.common.helpers
+
+import android.content.Context
+import androidx.core.content.ContextCompat
+import eu.kevin.common.R
+import eu.kevin.common.entities.KevinWebFrameColorsConfiguration
+import eu.kevin.common.extensions.getColorFromAttr
+import eu.kevin.common.extensions.isDarkMode
+import eu.kevin.common.extensions.toHexColor
+import eu.kevin.core.plugin.Kevin
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import java.util.*
+
+object WebFrameHelper {
+
+    fun getStyleAndLanguageQueryParameters(
```

Comment:
You don't need to supply `deviceLocale` anymore, because you have `Context` here. Also, it's misleading, because I would expect to get the query parameters with supplied locale, but it is not the case as we see below. Just pass the context to `getActiveLocaleCode` instead of `defaultLocale` and fallback to the extracted `Context` locale there.